src/component/tooltip/TooltipRichContent.ts

```diff
@@ -85,29 +85,17 @@ class TooltipRichContent {
         }
 
         const textStyleModel = tooltipModel.getModel('textStyle');
-
-        this.el = new ZRText({
-            style: {
-                rich: markupStyleCreator.richTextStyles,
+        const mergeTextStyle = createTextStyle(textStyleModel, {
                 text: content as string,
-                lineHeight: 22,
```

Comment:
`lineHeight` should use `22` to compat with old versions. Change this to `lineHeight: zrUtil.retrieve2(textStyleModel.get('lineHeight'), 22)`